src/lib.rs

```diff
@@ -39,7 +40,7 @@ impl Config {
             2 => match args[1].as_str() {
                 "fmt" => Ok(Self { mode: Mode::Format }),
                 "check" => Ok(Self { mode: Mode::Check }),
-                "version" => Ok(Self { mode: Mode::Version }),
+                "--version" | "-v" => Ok(Self { mode: Mode::Version }),
                 _ => Err("Unrecognized mode"),
```

Comment:
Just a nit.. since we're making a change right in this area.. how about in addition to "Unrecognized mode" we output what the recognized modes are for a little help?

Reply:
Done in faa074e, thanks!